salt/modules/win_pkg.py

```diff
@@ -1217,28 +1211,33 @@ def install(name=None, refresh=False, pkgs=None, **kwargs):
             ret[pkg_name] = 'Unable to locate package {0}'.format(pkg_name)
             continue
 
-        version_num = options.get('version', '')
+        version_num = options.get('version')
         #  Using the salt cmdline with version=5.3 might be interpreted
         #  as a float it must be converted to a string in order for
         #  string matching to work.
         if not isinstance(version_num, six.string_types) and version_num is not None:
             version_num = six.text_type(version_num)
 
+        # If the version was not passed, version_num will be None
         if not version_num:
             if pkg_name in old:
-                log.debug('A version (%s) already installed for package '
-                          '%s', version_num, pkg_name)
+                log.debug('pkg.install: \'%s\' version \'%s\' is already installed',
+                          pkg_name, old[pkg_name][0])
                 continue
-            # following can be version number or latest or Not Found
+            # Get the most recent version number available from winrepo.p
+            # May also return `latest` or an empty string
```

Comment:
or 'Not Found'    a package definition can contain Not Found to match when the version string in the app can not be found.